src/utils.ts

```
@@ -15,6 +15,11 @@ import {Inputs} from './inputs'
 
 const MINIMUM_GIT_VERSION = '2.18.0'
 
+const validateGitFetchArgs = (args: string[]): boolean => {
+  const forbiddenArgs = ['--upload-pack'];
```

ESLint

[error] Delete `;`

(prettier/prettier)